modelmapper/cleaner.py

```diff
@@ -162,34 +192,39 @@ def xls_date(x):
                 field_values[:] = map(lambda x: None if x is None else strptime(x, _format), field_values)
         return field_values
 
-    def clean(self, content_type, path=None, content=None, sheet_names=None):
+    def clean(self, content_type, path=None, content=None, sheet_names=None, ignore_new_fields=True):
```

Comment:
I think this should be `ignore_missing_fields` which is a broader category of what this is doing.

Reply:
👍